src/datepicker/datepicker.component.ts

```diff
@@ -231,7 +231,7 @@ export class DatePicker implements
 
 	protected _value = [];
 
-	protected _flatpickrOptions = {
+	protected _flatpickrOptions : flatpickr.Options.Options = {
```

Comment:
```suggestion
	protected _flatpickrOptions : Options = {
```

Additionally with this change, you should be able to remove the `@ts-ignore` comments that suppressed the type errors previously!